main.py

```diff
@@ -51,6 +53,11 @@ def reAward(fname):
 ##(/5) Task 3: CREATE code that will open a file and extract all the phone numbers 
 
 def rePhone(fname):
+  fhand = open(fname, 'r')
+  for line in fhand:
+    extr = re.findall('[0-9]{3,}-\S+', line)
```

Comment:
This will extract dates as well ('2008-01-09').
`extr = re.findall('\d{3}-\d{3}-\d{4}', line)`